infra/routes/layoutlm_inference_cache_generator/infra.py

```
@@ -25,14 +25,24 @@
+
+    Supports both single-pass and two-pass inference modes:
+    - Single-pass: Uses LAYOUTLM_TRAINING_BUCKET to auto-discover latest model
+    - Two-pass: Uses PASS1_MODEL_S3_URI + PASS2_MODEL_S3_URI or auto-discovery
+    """
 
     def __init__(
         self,
         name: str,
         *,
         layoutlm_training_bucket: Input[str],
         cache_bucket_name: Optional[Input[str]] = None,
```

Ruff (0.14.11)

Unused method argument: `cache_bucket_name`

(ARG002)